Makefile.vars

```diff
@@ -35,6 +35,7 @@ export AS                    # The assembler.
 export ASFLAGS               # Flags for the assembler.
 export LINK                  # The command used to link the files. Must take the same parameters as GCC, i.e. "ld" won't work.
 export LINKFLAGS             # Flags to supply in the linking step.
+export LTOFLAGS              # Pass link time optimization flags to submake
```

Comment:
I don't understand what this does from reading the comment in the Makefile. Is this a variant of CFLAGS?